e2e/queryit.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+"""
+QLever Query Tool for End2End Testing
+"""
+
+import sys
+import urllib.parse
+import urllib.request
+from typing import Dict, Any, List
+import json
+import yaml
+
+class BCol:
```

Comment:
This could use the builtin `Enum` class for proper enum like behaviour. (https://docs.python.org/3.6/library/enum.html)

Reply:
This has the problem that then I can no longer concatenate `BCol.BOLD` directly to a string because it would then convert to `"BCol.BOLD"`